DOL.WHD.Section14c.Log/ActionFilters/AuthorizationRequiredAttribute.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
```

Comment:
I would recommend not committing this at this time. It's not being used anywhere yet, but we'd want to make sure it doesn't accidentally get used since it would authorize any requests that had any value in a header named "Token"

Reply:
@klinden Let's sync up for your build issue.